### DISCUSSION TITLE
Localhost to Online

### DISCUSSION THREAD

**Discussion by VK2OZI** [magicbug/Cloudlog#986] [Q&A]
I have moved my Cloudlog from my localhost to an online server and have got the following error.

A Database Error Occurred

Error Number: 1146

Table 'tszhfkad_cloudlog.TABLE_HRD_CONTACTS_V01' doesn't exist

SELECT COUNT( * ) as count FROM TABLE_HRD_CONTACTS_V01 WHERE station_id = 1 AND COL_TIME_ON between '2021-04-09 00:00:00' AND '2021-04-09 23:59:59'

Filename: models/Logbook_model.php

Line Number: 984
 
Is anyone able to help?

**Comment by AndreasK79** [comment]
It seems to me that your database name is not the same. Did you change the config for the database in config/database.php?

**Reply by VK2OZI** [reply]
All parameters are correct, checked more than once. Not sure whats going on.

**Reply by AndreasK79** [reply]
Ok, and the table does exist in the online server?

**Reply by VK2OZI** [reply]
Yes, the same as it does in my XAMPP server locally

**Reply by VK2OZI** [reply]
I zipped and dumped the files, exported the SQL dumped it and changed DB name etc

**Reply by AndreasK79** [reply]
I managed to reproduce it locally here when I renamed the table. I got exactly the same error message. When Cloudlog can't find the logbook table in the database, it gives the error you described.

**Comment by VK2OZI** [comment]
![php-myadmin](https://user-images.githubusercontent.com/82219421/114299710-2a7be500-9b00-11eb-8886-27cea1ede458.jpg)

**Reply by VK2OZI** [reply]
for some reason when you dump the database on the live server the table HRD Contacts isn't in caps so you have to change in config to match. works now.
